### ISSUE TITLE
[REVIEW] [Accessibility]

### ISSUE THREAD

**Issue by domoritz** [journalovi/2025-sailynoja-visual-predictive-checks#8]
### Reviewed version

c97a8dc

### Perceivable

- [x] Contrast of all text and elements is sufficient. Geometries and large text must have >3:1 contrast against background. Regular text must have >4.5:1.
- [ ] Content has non-visual alternatives. All figures have alt text and videos have transcripts. Equations are screen reader accessible.
- [x] Font sizes for all texts are sufficiently large. Text must not be smaller than 9pt/12px in size. Ideally only minor text is rendered at 9pt (e.g., axis labels) while all other text is larger.
- [x] Where possible within reason, the article works even if I could not perceive colors. Colors should be augmented with patterns, textures, other visual cues, or tooltips if possible.
- [x] The article can be navigated comfortably with a screen reader. There are no unnecessary accessibility elements (e.g., axis ticks and labels) that clutter the screen reader experience.

### Screen reader

- [x] VoiceOver on macOS
- [ ] VoiceOver on iOS
- [ ] NVDA on Windows
- [ ] JAWS on Windows
- [ ] Not tested with a screen reader (explain below why not)

### Browser

- [x] Safari or other WebKit-based browser
- [ ] Chrome or other Chromium-based browser
- [ ] Firefox

### Comments on Perceivable

Alt texts are missing for all figures.
Please also disable tooltips for references as they are broken with screen readers. See https://github.com/quarto-dev/quarto-cli/issues/6500. 

### Operable

- [x] If there are interactive elements in the article, they can be operated with a keyboard.
- [x] If there are interactive elements in the article, there are textual instructions for how to use them or it's reasonable to assume that it's clear how to use them.
- [x] When navigating over the article with a keyboard, the focus indicator is always visible.
- [x] Interactions that work with a mouse also work on a touch screen (unless explicitly called out).

### Comments on Operable

All good.

### Understandable

- [x] All charts have a title and a description.
- [x] Changes in animated or interactive elements are easy to follow.
- [x] All axes have clear labels (either explicit or implicitly given the context).
- [x] There are legends for all charts that need them.

### Comments on Understandable

All good.

### Robust

- [x] All buttons use the `button` tag.
- [x] Headings, paragraphs, and figures are tagged correctly.

### Comments on Robust

All good.

### Assistive

- [x] The data density of all charts is appropriate. Use clustering or other data reduction techniques if elements compete for space but explain the method.
- [x] Navigation and interaction feels intuitive with all interaction methods (mouse, keyboard, touch).
- [x] All visually apparent features and relationships are described.
- [x] Formatting makes values human-readable. Use , or space to separate thousands. Align numbers to the right. Use the smallest, appropriate number of significant digits.

### Comments on Assistive

All good.

### Flexible

- [x] The article is readable on a phone.
- [x] The article can be zoomed and font-sizes change appropriately. The layout of the page does not break when zooming in or changing the font size (e.g., using CMD + or CMD - on macOS).
- [x] Long animations (if they exist) can be paused or stopped. Animations should be videos, not GIFs as they cannot be paused.
- [x] Style and charts use a consistent and familiar design (fonts, colors, etc). Interaction defaults should be consistent for all interactive elements.

### Comments on Flexible

All good. It's nice how the composed figures wrap automatically. 

### ORCID

0000-0002-3110-1053

**Comment by TeemuSailynoja** [comment]
Thank you, @chatchavan and @domoritz. I will add the missing alt-texts and disable the popup from the references as soon as possible. I hope the light box feature on the figures was not problematic with screen reader use, as I haven't got any personal experience in using one.

**Comment by TeemuSailynoja** [comment]
Hi @chatchavan, to make the workflow easiest for everyone, should I make a PR on per-issue basis and then ask the reviewer who requested the changes to review the PR?

**Comment by chatchavan** [comment]
If it's not much more work for you, you can do that. 

I suspect that each of the JoVI chairs (accessibility and open-practices) will use their requests as the starting point and test those aspects on the rendered version of the paper. So, the changes in the source code might not be seen.

**Comment by domoritz** [comment]
You can go ahead and merge the fixes. Please get familiar with the basics of a screen reader to test your article. Without some tight feedback loop, it'll be harder to get this right.

**Comment by domoritz** [comment]
Awesome. The alt texts look good.

**Comment by floe** [comment]
Reopening so it remains visible on the main issue page.
